Sources/ArgumentParserToolInfo/ToolInfo.swift

```diff
@@ -49,6 +49,8 @@ public struct CommandInfoV0: Codable, Hashable {
 
   /// Name used to invoke the command.
   public var commandName: String
+  /// List of command aliases.
+  public var aliases: [String]
```

Comment:
to maintain compatibility with tool info v1 we need to make this `Optional`

Reply:
Thanks! I'll make the change.

So I understand better (I'm new to the code, first PR), where would I need to look to know that? I just did a code search of the project for `aliases` and I didn't see any hints about this.

Is it just a rule that any new fields are Optional (it seems aliases was added later) to keep compatibility straightforward?